modules/oe_whitelabel_helper/src/Plugin/field_group/FieldGroupFormatter/MultiColumn.php

```diff
@@ -0,0 +1,266 @@
+      '#type' => 'textfield',
+      '#default_value' => $this->getSetting('first_column'),
+    ];
+    $form['second_column'] = [
+      '#title' => $this->t('Second column classes'),
+      '#description' => $this->t('Add custom classes to the second column.'),
+      '#type' => 'textfield',
+      '#default_value' => $this->getSetting('second_column'),
+    ];
+
+    $form['hide_table_if_empty'] = [
+      '#title' => $this->t('Hide the table if empty'),
+      '#description' => $this->t('Do not output any table or container markup if there are no rows with values.'),
+      '#type' => 'checkbox',
+      '#default_value' => $this->getSetting('hide_table_if_empty'),
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsSummary() {
+    $summary = parent::settingsSummary();
+    $summary[] = $this->t('Display results as a 2 column table.');
+
+    return $summary;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function preRender(&$element, $rendering_object) {
+    parent::preRender($element, $rendering_object);
+
+    $element['#mode'] = $this->context;
+    // Allow modules to alter the rows, useful for removing empty rows.
+    $children = Element::children($element, TRUE);
+    $this->moduleHandler->alter('field_group_table_rows', $element, $children);
+
+    if ($this->getSetting('hide_table_if_empty')) {
+      field_group_remove_empty_display_groups($element, []);
+      if ($element == []) {
+        return;
+      }
+    }
+
+    // Elements wrapper.
+    $element['#type'] = 'container';
+    $element['#attributes']['class'][] = 'my-3';
+
+    // Print the fieldset title.
+    $element['fieldset_title']['#type'] = 'markup';
+    $element['fieldset_title']['#markup'] = '<h4 class="fw-bold mb-4">' . $this->group->label . '</h4>';
+
+    // Fieldset data container.
+    $element['fieldset']['data']['#type'] = 'container';
+    $element['fieldset']['data']['#attributes']['class'][] = 'row';
+
+    // Set additional classes per column.
+    $classes = ['col-12', 'col-md-6'];
+    if ($this->getSetting('first_column')) {
+      $first_column_classes = array_merge($classes, [$this->getSetting('first_column')]);
+    }
+
+    if ($this->getSetting('second_column')) {
+      $second_column_classes = array_merge($classes, [$this->getSetting('second_column')]);
+    }
+    // Columns container.
+    $element['fieldset']['data']['first_column']['#type'] = 'container';
+    $element['fieldset']['data']['first_column']['#attributes'] = [
+      'class' => $this->getSetting('first_column') ? $first_column_classes : $classes,
+    ];
+    $element['fieldset']['data']['second_column']['#type'] = 'container';
+    $element['fieldset']['data']['second_column']['#attributes'] = [
+      'class' => $this->getSetting('second_column') ? $second_column_classes : $classes,
+    ];
+
+    // Number of items in the first columns.
+    $first_column_elements = array_slice($children, 0, ceil(count($children) / 2));
```

Comment:
I think ceil is waiting for a float not an int

Reply:
In fact ceil can receive both int and floats, but in this case ceil is rounding up for then we have un-even elements.